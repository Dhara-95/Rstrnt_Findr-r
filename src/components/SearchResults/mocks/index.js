import { RESTAURANT_BY_POSTCODE } from "../../../queries";

const mocks = [
  {
    request: {
      query: RESTAURANT_BY_POSTCODE,
      variables: { postcode: "W13 34H" },
    },
    result: {
      data: {
        dog: {
          name: "Douglas",
        },
      },
    },
  },
];

export { mocks };
