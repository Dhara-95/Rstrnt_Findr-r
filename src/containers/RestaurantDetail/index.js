import React from "react";

import { RestaurantLocationMap } from "../../components";

const RestaurantDetail = (props) => {
  const { restaurant } = props;
  console.log(restaurant);
  return (
    <>
      <h2 className="text-grey">Restaurant Detail page</h2>
      <RestaurantLocationMap />
    </>
  );
};

export { RestaurantDetail };
