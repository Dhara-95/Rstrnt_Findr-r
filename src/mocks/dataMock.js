const businesses = [
  {
    id: "OV4Lnl9UwUPi4W9tRwbqzA",
    name: "Apsley Chinese",
    categories: [{ title: "Fast Food" }],
    rating: 5.0,
    price: null,
    photos: [
      "https://s3-media2.fl.yelpcdn.com/bphoto/Bi7qfNYABueKOvQZsdhF3g/o.jpg",
    ],
    location: {
      address1: "28 London Road",
      address2: null,
      address3: null,
      city: "Hemel Hempstead",
      state: "HRT",
      postal_code: "HP3 9SB",
      country: "GB",
      formatted_address:
        "28 London Road\nHemel Hempstead HP3 9SB\nUnited Kingdom",
    },
    reviews: [
      {
        text:
          "Yummmmmmeeeee!\nFull sized ribs!\nTasty Singapore noodles.  \nPerfectly cooked chicken balls.\nTasty curry sauce. \nSpot on chow mien.",
        rating: 5,
      },
      {
        text:
          "We go here on occasion and love it. Recently refurbed but still very small inside. Love their curry sauce and always friendly staff.",
        rating: 5,
      },
    ],
    coordinates: { latitude: 51.7375787, longitude: -0.4688423 },
  },
  {
    id: "HGkMlvRYIaP0NCqiN4_nFQ",
    name: "Apsley Fish & Chips",
    categories: [{ title: "Fish & Chips" }],
    rating: 4.0,
    price: null,
    photos: ["https://s3-media3.fl.yelpcdn.com/bphoto/None/o.jpg"],
    location: {
      address1: "40 London Road",
      address2: null,
      address3: null,
      city: "Hemel Hempstead",
      state: "HRT",
      postal_code: "HP3 9SB",
      country: "GB",
      formatted_address:
        "40 London Road\nHemel Hempstead HP3 9SB\nUnited Kingdom",
    },
    reviews: [
      {
        text:
          "Great food. Friendly staff. Food portions reduced since credit crunch but still good quality.",
        rating: 4,
      },
    ],
    coordinates: { latitude: 51.7376153, longitude: -0.4689047 },
  },
  {
    id: "pNXAw1NcQEFTRylEI5rR_A",
    name: "Bull Rodizio",
    categories: [
      { title: "Steakhouses" },
      { title: "Brazilian" },
      { title: "Cocktail Bars" },
    ],
    rating: 4.5,
    price: null,
    photos: [
      "https://s3-media4.fl.yelpcdn.com/bphoto/v04pvvez9kgzYOSmD-lDSw/o.jpg",
    ],
    location: {
      address1: "62 London Rd",
      address2: "",
      address3: null,
      city: "Hemel Hempstead",
      state: "HRT",
      postal_code: "HP3",
      country: "GB",
      formatted_address: "62 London Rd\nHemel Hempstead HP3\nUnited Kingdom",
    },
    reviews: [
      {
        text:
          "Took a while to get seated although only one couple in front of us. \nOur table server was excellent, really spot on.\nThe salad buffet were served by staff...",
        rating: 3,
      },
      {
        text:
          "Excellent food and amazing service. I recommend the Picanha, the Beef, Garlic Beef, Sausage, Chicken, and most of all the grilled pineapples!!! Christina is...",
        rating: 5,
      },
      {
        text:
          "This place is amazing..best restaurant in this area. The buffet has such an amazing selection of vegetables and hot foods. Pao de queijo were fresh and...",
        rating: 5,
      },
    ],
    coordinates: { latitude: 51.738442, longitude: -0.470451 },
  },
  {
    id: "9UpIZckQmAAys9dHyttWHQ",
    name: "Raja Tandoori",
    categories: [{ title: "Indian" }, { title: "Pakistani" }],
    rating: 3.0,
    price: null,
    photos: [
      "https://s3-media3.fl.yelpcdn.com/bphoto/MyC_Qkx5GsR0m4rhukvh4g/o.jpg",
    ],
    location: {
      address1: "84 London Road",
      address2: "",
      address3: "",
      city: "Hemel Hempstead",
      state: "HRT",
      postal_code: "HP3 9SD",
      country: "GB",
      formatted_address:
        "84 London Road\nHemel Hempstead HP3 9SD\nUnited Kingdom",
    },
    reviews: [
      {
        text:
          "More than happy to order it from here again.\n\nWe've ordered a few times once it was quite runny, with a lot of sauce, but no grease, this time there was...",
        rating: 3,
      },
    ],
    coordinates: { latitude: 51.7389005, longitude: -0.4712796 },
  },
  {
    id: "BoOlYngEdTL4Gw_66ikCMw",
    name: "Efeler Kebabs",
    categories: [{ title: "Fast Food" }],
    rating: 4.5,
    price: null,
    photos: ["https://s3-media3.fl.yelpcdn.com/bphoto/None/o.jpg"],
    location: {
      address1: "98 London Road",
      address2: "",
      address3: "",
      city: "Hemel Hempstead",
      state: "HRT",
      postal_code: "HP3 9SD",
      country: "GB",
      formatted_address:
        "98 London Road\nHemel Hempstead HP3 9SD\nUnited Kingdom",
    },
    reviews: [
      {
        text:
          "I have been ordering takeaways from these guys for quite a time now. They are efficient, friendly and incredibly QUICK with the deliveries!!!\nIf you are...",
        rating: 5,
      },
      {
        text:
          "Fantastic late-night takeaway with great pita and remarkable chips.  The pizza is alright but not spectacular.   So good and open so late.   And free...",
        rating: 4,
      },
    ],
    coordinates: { latitude: 51.7390343, longitude: -0.4714636 },
  },
  {
    id: "tseC8EwTL7HyxtXc3IdICw",
    name: "McDonald's",
    categories: [{ title: "Burgers" }, { title: "Fast Food" }],
    rating: 1.0,
    price: null,
    photos: ["https://s3-media3.fl.yelpcdn.com/bphoto/None/o.jpg"],
    location: {
      address1: "The Bell",
      address2: "241 London Road",
      address3: "Apsley",
      city: "Hemel Hempstead",
      state: "HRT",
      postal_code: "HP3 9SE",
      country: "GB",
      formatted_address:
        "The Bell\n241 London Road\nApsley\nHemel Hempstead HP3 9SE\nUnited Kingdom",
    },
    reviews: [
      {
        text:
          "Sad to say but these guys get orders wrong all the time. Errors are made from new starters right through to managers.",
        rating: 1,
      },
    ],
    coordinates: {
      latitude: 51.7394222987057,
      longitude: -0.472690451969694,
    },
  },
  {
    id: "9T9VDBHBMPR_voBN2STiDg",
    name: "The Paper Mill",
    categories: [{ title: "British" }, { title: "Pubs" }],
    rating: 3.5,
    price: "\u00a3\u00a3",
    photos: [
      "https://s3-media4.fl.yelpcdn.com/bphoto/oCfi6ypRg6inFjsZQwqyIg/o.jpg",
    ],
    location: {
      address1: "Stationers Place",
      address2: "Apsley",
      address3: "",
      city: "Hemel Hempstead",
      state: "HRT",
      postal_code: "HP3 9RH",
      country: "GB",
      formatted_address:
        "Stationers Place\nApsley\nHemel Hempstead HP3 9RH\nUnited Kingdom",
    },
    reviews: [
      {
        text:
          "The atmosphere was good. The bar tenders were good. The beer was good. The food, not so great. This is definitely a great place for drinks being so close to...",
        rating: 3,
      },
      {
        text:
          "Set in the expensive Apsley lock area, this place attracts an affluent crowd. The priced are quite high, but drinks are good here. Usually a wheat beer on...",
        rating: 5,
      },
      {
        text:
          "nice looking pub, gets busy and can sometimes be a bit of a chore getting a table downstairs in the bar area if you just want a drink and a snack. there is...",
        rating: 4,
      },
    ],
    coordinates: { latitude: 51.733304, longitude: -0.4616731 },
  },
];

export { businesses };
