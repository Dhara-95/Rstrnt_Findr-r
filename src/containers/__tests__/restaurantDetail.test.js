import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";

import { RestaurantDetail } from "../RestaurantDetail";
import { businesses } from "../../mocks/dataMock";

it("displays the details of a single restaurant correctly", async () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <RestaurantDetail restaurant={businesses[0]} postcode={"W13 3RH"} />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
