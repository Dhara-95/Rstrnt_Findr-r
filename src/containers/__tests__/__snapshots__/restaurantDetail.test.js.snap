// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the details of a single restaurant correctly 1`] = `
Array [
  <div
    className="pb-16"
  >
    <a
      className="text-grey"
      href="/searchresults"
      onClick={[Function]}
    >
      &lt;&lt; Back to see all restaurants for W13 3RH
    </a>
  </div>,
  <h2
    className="restaurant-name text-white text-4xl"
  >
    Apsley Chinese
  </h2>,
  <div
    className="flex mx-auto w-full h-80 py-8 overflow-hidden relative justify-center content-center lg:w-2/3 xl:w-2/3 2xl:w-2/3 "
  >
    <img
      alt="Apsley Chinese"
      className="w-full h-auto block object-cover"
      src="https://s3-media2.fl.yelpcdn.com/bphoto/Bi7qfNYABueKOvQZsdhF3g/o.jpg"
    />
  </div>,
  <div
    className="pb-4"
  >
    <div
      className="restaurant-tag"
    >
      <span
        className="text-grey"
      >
        Fast Food
      </span>
    </div>
  </div>,
  <h2
    className=" restaurant-price text-grey"
  >
    No price rating available
  </h2>,
  <div
    className="py-8"
  >
    <div
      className="mx-auto"
    />
  </div>,
  <h3
    className="text-grey text-xl"
  >
    Overall Rating
  </h3>,
  <div
    className="restaurant-rating"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-star fa-w-18 fa-lg "
      color="gold"
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-star fa-w-18 fa-lg "
      color="gold"
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-star fa-w-18 fa-lg "
      color="gold"
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-star fa-w-18 fa-lg "
      color="gold"
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-star fa-w-18 fa-lg "
      color="gold"
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>,
  <div>
    <div
      className="text-grey w-full mx-auto py-4 lg:w-1/2 xl:w-1/2 2xl:w-1/2"
    >
      <div
        className="flex justify-between"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-quote-left fa-w-16 fa-lg "
          color="light-grey"
          data-icon="quote-left"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M464 256h-80v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8c-88.4 0-160 71.6-160 160v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48zm-288 0H96v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8C71.6 32 0 103.6 0 192v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-quote-right fa-w-16 fa-lg "
          color="light-grey"
          data-icon="quote-right"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <div
        className="w-4/5 mx-auto"
      >
        <p
          className="text-grey text-xl text-center font-bold"
        >
          Yummmmmmeeeee!
Full sized ribs!
Tasty Singapore noodles.  
Perfectly cooked chicken balls.
Tasty curry sauce. 
Spot on chow mien.
        </p>
        <div
          className="flex flex-col items-end"
        >
          <div
            className="restaurant-rating"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <p
            className="text-grey text-xl"
          >
            — Louie-Mae B.
          </p>
        </div>
      </div>
    </div>
    <div
      className="text-grey w-full mx-auto py-4 lg:w-1/2 xl:w-1/2 2xl:w-1/2"
    >
      <div
        className="flex justify-between"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-quote-left fa-w-16 fa-lg "
          color="light-grey"
          data-icon="quote-left"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M464 256h-80v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8c-88.4 0-160 71.6-160 160v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48zm-288 0H96v-64c0-35.3 28.7-64 64-64h8c13.3 0 24-10.7 24-24V56c0-13.3-10.7-24-24-24h-8C71.6 32 0 103.6 0 192v240c0 26.5 21.5 48 48 48h128c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-quote-right fa-w-16 fa-lg "
          color="light-grey"
          data-icon="quote-right"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <div
        className="w-4/5 mx-auto"
      >
        <p
          className="text-grey text-xl text-center font-bold"
        >
          We go here on occasion and love it. Recently refurbed but still very small inside. Love their curry sauce and always friendly staff.
        </p>
        <div
          className="flex flex-col items-end"
        >
          <div
            className="restaurant-rating"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-star fa-w-18 fa-lg "
              color="gold"
              data-icon="star"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
          <p
            className="text-grey text-xl"
          >
            — jasondob1
          </p>
        </div>
      </div>
    </div>
  </div>,
]
`;
